os/arch/arm/src/Makefile

```diff
@@ -243,5 +247,6 @@ ifeq ($(BOARDMAKE),y)
 endif
 	$(call DELFILE, Make.dep)
 	$(call DELFILE, .depend)
+	$(call DELFILE, $(COBJS))
```

Comment:
Duplicated. `$(call CLEAN)` does this. Please revert this line.

Reply:
Reverted in latest commit